source/views/dictionary/list.js

```diff
@@ -77,21 +99,43 @@ export class DictionaryView extends React.Component {
     return <ListSeparator key={`${sectionId}-${rowId}`} />
   }
 
+  handleResults = (results: Object) => {
+    this.setState({results: results})
+  }
+
   render() {
+    if (!terms) {
+      return <LoadingView />
```

Comment:
Why show the loading spinner? If there are no definitions, they'll never show up.